static/js/map.js

```diff
@@ -20,6 +20,7 @@ function initMap() {
     type: "GET",
     data: {
       "$limit" : 5000,
+      // If this is an API key we don't want this in a repo ...
```

Comment:
I can show you how we secure sensitive data with Javascript. It seems like we may need to here.